client/src/api/supplyRoutesSlice.ts

```diff
@@ -1,30 +1,52 @@
 import { RootState } from "../app/store";
 import { gameLoaded, gameUnloaded } from "./actions";
 import { SupplyRoute } from "./liberationApi";
-import { createSlice } from "@reduxjs/toolkit";
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
 
 interface SupplyRoutesState {
-  routes: SupplyRoute[];
+  routes: { [id: string]: SupplyRoute };
 }
 
 const initialState: SupplyRoutesState = {
-  routes: [],
+  routes: {},
 };
 
 export const supplyRoutesSlice = createSlice({
   name: "supplyRoutes",
   initialState,
-  reducers: {},
+  reducers: {
+    updated: (state, action: PayloadAction<SupplyRoute[]>) => {
+      for (const route of action.payload) {
+        const id = route.id;
+        let points = route.points;
+        if (points.length > 0) {
```

Comment:
Isn't this always true?

Reply:
Relates to deleting the points in back-end...